learn/experimental/autobatching.md

```diff
@@ -0,0 +1,34 @@
+# Auto-batching
+
+If enabled, this feature automatically combines multiple document addition requests ([`POST`](/reference/api/documents.md#add-or-replace-documents) or [`PUT`](/reference/api/documents.md#add-or-update-documents)) into batches to speed up indexation.
```

Comment:
Pretty obvious, but I think we should mention it anyway: if one update in a batch fails, the entire batch fails